tests/test_faction_denunciation_627.py

```
@@ -0,0 +1,628 @@
+        "faction_situations",
+        "character_personas",
+    ):
+        assert key in facts, key
+        assert isinstance(facts[key], list), key
+
+    assert any(int(d["dossier_id"]) == did for d in facts["forked_dossiers"])
+    assert facts["faction_enmities"], "须有敌对关系事实"
+    assert facts["faction_situations"], "须有派系处境定性档"
+    assert facts["character_personas"], "须有人物个性"
+
+    # 处境/个性为定性档，非裸分
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)